cmake/modules/shared/CompilerSettings.cmake

```diff
@@ -8,14 +8,14 @@
 
 # Settings intended to be applied to every Swift target in this package.
 # Analogous to project-level build settings in an Xcode project.
+add_compile_options(
+  "SHELL:$<$<COMPILE_LANGUAGE:Swift>:-swift-version 6>")
```

Comment:
I think that this is better spelt as the following at the top level CMakeLists.txt:

```cmake
set(Swift_LANGUAGE_VERSION 6)
```

Reply:
Looks like it's spelled **CMAKE**_Swift_LANGUAGE_VERSION, but that does work!